conftest.py

```diff
@@ -0,0 +1,21 @@
+from functools import lru_cache
+from pathlib import Path
+
+import pytest
+
+
+XML_ROOT = Path("tests/xml")
+
+
+@pytest.fixture()
+@lru_cache()
+def saml_request_minimal() -> str:
+    with (XML_ROOT / "min/request/sample_saml_request_minimal.xml").open("r") as f:
+        return f.read()
+
+
+@pytest.fixture()
+@lru_cache()
```

Comment:
This caches the string being returned so that the fixture doesn't open the file every time it is being used.